examples/README.md

```diff
@@ -4,7 +4,7 @@ This is a quick example about how you use Pangea Go SDK, set up and run it.
 
 ## Setup
 
-Set up environment variables ([Instructions](https://pangea.cloud/docs/getting-started/integrate/#set-environment-variables)) `PANGEA_AUDIT_TOKEN` and `PANGEA_DOMAIN` with your project token configured on Pangea User Console (token should have access to Audit service [Instructions](https://pangea.cloud/docs/getting-started/configure-services/#configure-a-pangea-service)) and with your pangea domain.
+Set up environment variables ([Instructions](https://pangea.cloud/docs/getting-started/integrate/#set-environment-variables)) `PANGEA_<service>_TOKEN` and `PANGEA_DOMAIN` with your project token configured on Pangea User Console (token should have access to <service> [Instructions](https://pangea.cloud/docs/getting-started/configure-services/#configure-a-pangea-service)) and with your pangea domain.
```

Comment:
Note that the second `<service>` does not render on GitHub.
![image](https://github.com/pangeacyber/pangea-go/assets/733364/9678e994-cda3-40a5-b8b3-c8f537fbeb03)

Reply:
Updated merging main